packages/avatars/src/styled/utility.ts

```diff
@@ -6,16 +6,26 @@
  */
 
 import { getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { ThemeProps, DefaultTheme } from 'styled-components';
+import { math } from 'polished';
 
-import { STATUS } from '../types';
+import { STATUS, SIZE } from '../types';
 
+export const TRANSITION_DURATION = 0.25;
+
+const [xxs, xs, s, m, l] = SIZE;
 const [active, available, away, transfers, offline] = ['active', ...STATUS];
 
+export interface IStyledStatusIndicatorProps extends ThemeProps<DefaultTheme> {
+  readonly size?: typeof SIZE[number];
+  readonly type?: typeof STATUS[number] | 'active';
+}
+
 export function getStatusColor(
-  status: typeof STATUS[number] | 'active' | undefined,
-  theme = DEFAULT_THEME
+  type?: IStyledStatusIndicatorProps['type'],
+  theme?: typeof DEFAULT_THEME
```

Comment:
```suggestion
  theme?: ThemeProps<DefaultTheme>
```

Reply:
I think `ThemeProps<DefaultTheme>` would work here if `{ theme}` was a property on an object (AKA props). I replaced this with `IStyledStatusIndicatorProps['theme']`.